diff --git a/src/android/cc/fovea/PurchasePlugin.java b/src/android/cc/fovea/PurchasePlugin.java
index a50c89859f4f4ca73f7f6a222736146bbf92702d..dbf22ee4eb7dc57a46ea27f1952e62cd8151d9bd 100644
--- a/src/android/cc/fovea/PurchasePlugin.java
+++ b/src/android/cc/fovea/PurchasePlugin.java
@@ -665,7 +665,7 @@ public final class PurchasePlugin
 
   private Purchase findPurchaseByProductId(final String productId) {
     for (Purchase p : mPurchases) {
-      if (p.getSkus().contains(productId))
+      if (p.getProducts().contains(productId))
         return p;
     }
     return null;
